lib/Minz/Request.php

```diff
@@ -307,8 +359,41 @@ private static function magicQuotesOff() {
 		}
 	}
 
+	/**
+	 * @return array
+	 */
+	private static function extractHeaders() {
```

Comment:
I still cannot see the reason for doing such a copy, and doing so with a manual copy